checker/lib/README

```diff
@@ -3,20 +3,25 @@ This directory contains annotated versions of libraries.
 
 Instructions for users:
 
-During type-checking, put these versions on your classpath,
+During type-checking, put these jars on your classpath,
 to obtain more precise type-checking with fewer false positive warnings.
 
+If the file for the library you need is a stub file (its extension is
```

Comment:
Shouldn't this be automatically handle by the checker, unless the user has created a new .astub file?
I would note that.